modules/storage.js

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable no-unused-vars */
 class Storage {
```

Comment:
- [ ] When I reload the page The books are gone please make sure that the books are stored and retrieved from local storage